src/main/asciidoc/_chapters/ops_mgt.adoc

```diff
@@ -1738,6 +1738,83 @@ hbase.regionserver.authenticationFailures::
 hbase.regionserver.mutationsWithoutWALCount ::
   Count of writes submitted with a flag indicating they should bypass the write ahead log
 
+[[rs_meta_metrics]]
+=== Meta Table Load Metrics
+
+HBase meta table metrics collection feature is available in HBase 1.4+ but it is disabled by default, as it can
```

Comment:
Actually, it's available since 1.3
Could you please double check that? thanks

Reply:
Thanks, I just checked it. I don't find the MetaTableMetrics coprocessor or the lossy counting related config parameter in 1.3.
Also the whole metrics system was changed between 1.3 and 1.4 (you don't have the `hbase-metrics` maven module in 1.3), I guess this is why it wasn't merged back from 1.4.